.github/workflows/run_mmperf.yml

```diff
@@ -0,0 +1,93 @@
+name: mmperf
```

Comment:
Can you add a comment to this file as well with some information about what this workflow does? There should be enough information here for someone who forks the IREE repo to know if they want to run this workflow.

(and you should _technically_ add a license header)

Reply:
Added